### ISSUE TITLE
Extended barcodes.csv files not accepted

### ISSUE THREAD

**Issue by as1802** [CorwinAnsley/artifice#23]
Hi, I've been trying the extended barcodes.csv files (where we add more columns than sample and barcode e.g. Institute, EPID) but this kicks up an error that "sample" or "barcode" headers cannot be found. Is there a check in ARTIFICE that the would stop more columns being added? We have PIRANHA v1.0.6 loaded, which should be able to handle these extended files. 
("`sample` must be a column name in barcode csv file")
Thanks!

**Comment by as1802** [comment]
Got it, I think there was an invisible difference in the file formatting that was tripping this as an error in my extended file. I copied over "sample" and "barcode" from another file and all is fine.

**Comment by aineniamh** [comment]
Yeah, sometimes excel or other spreadsheet software adds in formatting invisibles to a file- if you open the file in the terminal it often shows up these things. Alternatively what you did (delete the row and write again in a plain text editor) works too!
